### PR TITLE
GCS:Welcome: Add Autotown results to panel

### PR DESCRIPTION
**Pull request by tracernz**
Part of #73
screenshot from 2016-02-22 22 02 34

[Review on Reviewable](https://reviewable.io/reviews/d-ronin/dronin/678)

### CODE REVIEW

File: ground/gcs/share/welcome/CommunityPanel.qml [hunk 1]
```diff
@@ -18,7 +18,18 @@ Item {
         anchors.left: parent.left
         anchors.top: parent.top
         anchors.bottom: parent.bottom
-        width: parent.width*0.6
+        width: parent.width*0.5
```
Comment (mluessi) [inline ground/gcs/share/welcome/CommunityPanel.qml]: spacing before/after *

File: ground/gcs/share/welcome/CommunityPanel.qml [hunk 2]
```diff
@@ -18,7 +18,18 @@ Item {
...
+
+        onClicked: welcomePlugin.openUrl(url)
+    }
+
+    AutoTunePanel {
+        id: autoTunePanel
+        anchors.left: gitHubNewsPanel.right
+        anchors.top: parent.top
+        anchors.bottom: parent.bottom
+        width: parent.width*0.25
```
Comment (mluessi) [inline ground/gcs/share/welcome/CommunityPanel.qml]: same here

**Comment by mluessi** [comment]
Very nice. Code looks good. Will test later today.

**Comment by mlyle** [comment]
Very nice.  @dustin doing something like this makes it more important that autotown discard garbage so we put something meaningful on the screen-- have you/we done anything in that area?

**Comment by dustin** [comment]
There's not been all that much garbage.  Multiple tunes collapse into one.
There are a few too low or too high ones.  I could make it possible to
filter those out.

**Comment by tracernz** [comment]
Review comments addressed plus another couple of things I noticed myself.

**Comment by dustin** [comment]
Seems like a small issue here:

image

**Comment by tracernz** [comment]
Damn, did I just break it -_-

**Comment by tracernz** [comment]
Fixed :/.

**Comment by dustin** [comment]
:lgtm: 

---

Review status: 0 of 8 files reviewed at latest revision, 2 unresolved discussions.

---

_Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/d-ronin/dronin/678#-:-KBBq05g6xLjyLeKnhvG:980077423)_

**Pull request merged**